src/loaddb/load_server_loader.cpp

```diff
@@ -118,6 +124,13 @@ namespace cubload
 	return;
       }
 
+    // Check if we have to ignore this class.
+    if (is_class_ignored (class_name, true))
```

Comment:
is_class_ignored looks like a function that does a check, but it does more. it is more clear if the code is something like this:
```c
if (is_class_ignored (class_name))
  {
    m_session.append_log_msg (LOADDB_MSG_IGNORED_CLASS, class_name);
    class_entry *cls_entry = new class_entry (class_name, m_clsid, true);
    m_session.get_class_registry ().register_ignored_class (cls_entry, m_clsid);
    return;
  }
```